doc/playbook.rst

```diff
@@ -461,6 +461,9 @@ The server is visible from ``rubin-devl``, and can be accessed through, e.g.,
 
    psql -h usdf-prompt-processing-dev.slac.stanford.edu lsst-devl rubin
 
+Credentials
+-----------
+
 For passwordless login, create a ``~/.pgpass`` file with contents:
```

Comment:
Should there be a "Postgres" header here?

Reply:
I was unsure of the best choice, since the Postgres credentials include those for the Butler, which are needed even if you are using Cassandra. I could separate out the one line that is for the Postgres APDB, but that seems awkward to split apart the `~/.pgpass` contents. I could add a sentence to the Cassandra APDB section noting that the pgpass file is still needed for the Butler.

Reply:
That's not actually true -- Butler can take credentials from either source (and in DM-44141 we moved ours to db-auth).

Reply:
OK, I added the Postgres header and added a comment in the Cassandra section about Butler credentials.